datafusion/core/src/physical_plan/file_format/avro.rs

```diff
@@ -24,17 +24,23 @@ use crate::physical_plan::{
 use arrow::datatypes::SchemaRef;
 
 use crate::execution::context::TaskContext;
+#[cfg(feature = "avro")]
```

Comment:
Isn't this whole file `#[cfg(feature = "avro")]` ?

In other words, would this work?

```suggestion
```